src/DriveShare.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
```

Comment:
```suggestion
namespace Owncloud\OcisPhpSdk;
```

Probably needs the namespace to be set.